src/language.rs

```diff
@@ -46,6 +47,7 @@ const EXTENSIONS: &[(Language, &[&str])] = &[
     (Language::OcamlInterface, &["mli"]),
     (Language::Rust, &["rs"]),
     (Language::Toml, &["toml"]),
+    (Language::TreeSitterQuery, &["scm"]),
```

Comment:
Maybe we should move away from the `.scm` extension. Tree-sitter query files are Scheme-like, but not Scheme proper, and by squatting on the Scheme extension we limit ourselves if/when we want to format actual Scheme.

My personal preference would be `.topiary` for our query files. While they are generic Tree-sitter queries -- so, perhaps `.tsq` would suffice -- they contain Topiary-specific predicates. Perhaps, therefore:

```suggestion
    (Language::TreeSitterQuery, &["topiary", "tsq"]),
```

...with the appropriate changes to the `README` and elsewhere, plus renaming of everything under the `languages` directory.

Reply:
I agree on your point, I made it myself while writing this.
`.topiary` and `.tsq` both look right to me

Reply:
I also raised the point on Slack, to see what the rest of the team think.

Reply:
My single issue with switching away from `.scm` is that we would go against what is common. Even the "official" tree-sitter documentation explicitly uses `.scm`.

> and by squatting on the Scheme extension we limit ourselves if/when we want to format actual Scheme.

Yeah, but this is not a solution. Topiary should still be able to format the tree-sitter query files from other projects, e.g. Neovim, helix, emacs, and thus it will always encounter `.scm` files that are actually query files. Renaming our query files to `.tsq` will ensure that we know when we are formatting our own query files, but not from these other projects.

Reply:
I wasn't aware how entrenched `.scm` was in the Tree-sitter ecosystem. That does change things...

(Also, devil's advocate: Scheme is traditionally formatted with both indents and alignments on s-expressions...which Topiary doesn't [[currently](https://github.com/tweag/topiary/issues/170)] support.)

Reply:
I'm currently leaning towards leaving it as it is right now. If we ever decide to format proper Scheme files, the refactoring will be minimal anyway